receiver/prometheusreceiver/internal/transaction.go

```diff
@@ -142,8 +142,11 @@ func (tr *transaction) Commit() error {
 	}
 	// Note: metrics could be empty after adjustment, which needs to be checked before passing it on to ConsumeMetricsData()
 	if tr.jobsMap != nil {
-		metrics = NewMetricsAdjuster(tr.jobsMap.get(tr.job, tr.instance), tr.logger).AdjustMetrics(metrics)
+		dropped := 0
+		metrics, dropped = NewMetricsAdjuster(tr.jobsMap.get(tr.job, tr.instance), tr.logger).AdjustMetrics(metrics)
+		droppedTimeseries += dropped
 	}
+	observability.RecordMetricsForMetricsReceiver(tr.ctx, numTimeseries, droppedTimeseries)
```

Comment:
Related to previous q: is it worth to have a z-page showing init/reset breakdown?

Reply:
Similarly to previous, If this was a general property of receivers it would be but it's specific to Prometheus so I don't think it's something to expose generally.